OPHD/Common.h

```diff
@@ -367,3 +367,9 @@ int pullTruckFromInventory();
  * \return 1 on success, 0 otherwise.
  */
 int pushTruckIntoInventory();
+
+
+const auto formatDiff = [](int diff)
+{
+	return ((diff > 0) ? "+" : "") + std::to_string(diff);
+};
```

Comment:
... or make them global functions.

In which case, these should probably be functions, rather than lambdas.

Reply:
Common is 'global'. I figured it's such a basic thing that it could easily be in-lined into wherever they need to be. I suppose the compiler can figure that out on its own with an `inline` hint but eh. Again, more practice with lambda syntax.